CHANGELOG.md

```diff
@@ -8,10 +8,18 @@ The Twilio Programmable Video SDKs use [Semantic Versioning](http://www.semver.o
 New Features
 ------------
 
-- You can now import type definitions for the SDK APIs to your project as shown below. (JSDK-3007)
+- You can now import type definitions for the SDK APIs to your project as shown below. Twilio-Video types will take priority when installed at the same time as the Definitely Typed type definitions. (JSDK-3007)
```

Comment:
I suggest something like:
twilio-video now includes typescript definitions for the SDK APIs. Previously typescript developers relied on [definitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/1a7a99db8ec25d48f3dfec146af742e5bc40a5f7/types/twilio-video/index.d.ts) for these definitions. We would like to thank folks at DefinitelyTyped for maintaining these definitions. Going forward these will be included in the library and will take precedence over typescript definitions. You can now remove definitelyTyped dependency from your project.

Reply:
@makarandp0 I like this! @manjeshbhargav Should I also include an acknowledgement to DefinitelyTyped in the changelog?